index.html

```diff
@@ -0,0 +1,216 @@
+            <h1 class="h1-global">Sharm el-Sheikh Climate Change Conference</h1>
+            <h2 class="h2">creative commons Climate Change summit 2022</h2>
+            <hr class="hr-about-15">
+            <p class="article-1-about">The 2022 United Nations Climate Change Conference, more commonly referred to as
+                COP27, will be the 27th United Nations Climate Change conference, to be held from 6 to 18 November 2022
+                in Sharm El Sheikh, Egypt, the Government of the Arab Republic of Egypt will host the 27th Conference of
+                the Parties of the UNFCCC (COP 27), with a view to building on previous successes and paving the way for
+                future ambition to effectively tackle the global challenge of climate change. </p>
+            <p class=" date-about">Please Contact us per Email for any further questions
+                about Sharm el-Sheikh Climate Change Conference
+            </p>
+            <p class=" place-about">info@cop27.eg</p>
+        </article>
+    </section>
+    <section>
+        <h1 class="h1 h1-about">creative commons
+            Sharm el-Sheikh Climate Change Conference 2022 Logo
+        </h1>
+        <hr class="hr-about-2">
+        <p class="articl-2">The globe incorporated in the logo represents how we should contain earth and keep it
+            preserved, balanced and safe. It also represents Africa, the continent hosting COP27 in 2022. </p>
+        <img class="art-logo" src="img/logo1.png" alt="cop-27-logo">
+        <h1 class="h1 h1-about">See the past climate change confrence</h1>
+        <hr class="hr-about-4">
+        <p class="articl-2">Take a look at the last climate change conference Lorem ipsum dolor sit amet Lorem, ipsum
+            dolor.</p>
+        <div class="img-big-container">
+            <div class="img-container">
+                <img class="cop26-1" src="img/cop-2-cop26-2.png" alt="cop-26 photo">
+                <div class="center">United Nations climate summit UK</div>
+            </div>
+            <div class="img-container">
+                <img class="cop26-1" src="img/cop-1-26-1-960x640.png" alt="cop-26 photo">
+                <div class="center">United Nations climate summit Madrid</div>
+            </div>
+        </div>
+    </section>
+    <footer>
+        <h3 class="footer-h">Partner</h3>
+        <hr class="hr-about">
+        <div class="footer-container">
+            <img class="footer-img" src="img/air.png" alt="un">
+            <img class="footer-img" src="img/airbnb.png" alt="airbnb">
+            <img class="footer-img" src="img/daumkakao.PNG" alt="daumkakao">
+            <img class="footer-img" src="img/google.png" alt="google">
+            <img class="footer-img" src="img/mozilla.png" alt="mozilla">
+            <img class="footer-img" src="img/naver.PNG" alt="naver">
+            <img class="footer-img" src="img/egy.png" alt="Egypt air logo">
+        </div>
```

Comment:
- [ ] Please wrap this in a `ul` tag and wrap each image in a `li` tag to make the code semantic. A list of similar items is best wrapped in a `ul` tag.

Reply:
@Lucas-Masaba Thanks a lot for your efforts in reviewing my code, 
I really appreciated it. 🙂